<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerClass</class>
 <widget class="QMainWindow" name="ControllerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>182</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Controller</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStep"/>
   <addaction name="actionClear"/>
   <addaction name="actionInitialize"/>
  </widget>
  <widget class="QStatusBar" name="statusBarGeneration"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>229</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Simulation">
    <property name="title">
     <string>&amp;Simulation</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="actionStep"/>
    <addaction name="actionClear"/>
    <addaction name="actionInitialize"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menu_Simulation"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="controller.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Start the simulaiton</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="controller.qrc">
     <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause the simulation</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="controller.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="toolTip">
    <string>Advance the simulation one time step</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear the lattice</string>
   </property>
  </action>
  <action name="actionInitialize">
   <property name="text">
    <string>Initialize</string>
   </property>
   <property name="toolTip">
    <string>Intialize the lattice's cells</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Open settings dialog</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="controller.qrc"/>
 </resources>
 <connections/>
</ui>
